artsy/force
# feat: Add new navbar and download app banner in footer

https://artsyproduct.atlassian.net/browse/GRO-156

This PR adds the new navbar and the download app banner in the footer.

## What changed

- Adds new navbar and new download app banner in the footer
- Many small styling tweaks, especially for legacy Force pages
- Updates styling for flyouts to look good at all viewports
- Updates some basic styling attributes on legacy pages to match v2 pages (e.g. `default-page-width`)

## QA plan

This work is currently in a review app at https://new-navbar.artsy.net.

See QA doc for list of pages to check: https://www.notion.so/artsy/Nav-Bar-QA-ef1419c4644d46728f30b2ae47c24785

## Screenshots

![Screen Shot 2021-02-11 at 3 14 04 PM](https://user-images.githubusercontent.com/4432348/107693955-c7dfa780-6c7c-11eb-9414-f08f3e0cd263.png)

![Screen Shot 2021-02-11 at 3 14 12 PM](https://user-images.githubusercontent.com/4432348/107693964-c8783e00-6c7c-11eb-96a4-5b56d339d3d8.png)

![Screen Shot 2021-02-11 at 3 17 47 PM](https://user-images.githubusercontent.com/4432348/107693963-c8783e00-6c7c-11eb-98fc-7cbff10beb89.png)

![Screen Shot 2021-02-11 at 3 18 04 PM](https://user-images.githubusercontent.com/4432348/107693961-c7dfa780-6c7c-11eb-8b64-9904482e2e9f.png)

![Screen Shot 2021-02-11 at 3 18 30 PM](https://user-images.githubusercontent.com/4432348/107693960-c7dfa780-6c7c-11eb-996b-0c821e0540d2.png)

cc @artsy/grow-devs
